scripts/linux/trcE2ETest.sh

```diff
@@ -894,7 +894,7 @@ E2E_TEST_DIR="${E2E_TEST_DIR:-$(pwd)}"
 DEPLOYMENT_TEST_UPDATE_PERIOD="${DEPLOYMENT_TEST_UPDATE_PERIOD:-00:03:00}"
 EVENT_HUB_CONSUMER_GROUP_ID=${EVENT_HUB_CONSUMER_GROUP_ID:-\$Default}
 EDGE_RUNTIME_BUILD_NUMBER=${EDGE_RUNTIME_BUILD_NUMBER:-$ARTIFACT_IMAGE_BUILD_NUMBER}
-TEST_START_DELAY="${TEST_START_DELAY:-00:02:00}"
```

Comment:
So you want all the arch to have the delay of 6 minutes even AMD64?

Reply:
The problem was observed on AMD64. And I think that is the only arch this script is called by. There is logic for arm, but afaik we don't use it.